ads/how-to-guides/scheduling-breaks.md

```diff
@@ -111,6 +113,10 @@ The body of the post request can have the following properties:
 - `source` is the optional identifier of the source for which the ad needs to be scheduled. The signaling service will only insert the ad for playlists containing part of this in their URL. If not set the ad is scheduled in every playlist.
 - `layout` is the optional OptiView Ads layout, which can be set to `SINGLE`, `DOUBLE`, `LSHAPE_AD` or `LSHAPE_CONTENT`. If it is `undefined` the default layout of the signaling service will be chosen.
 - `assetURI` is the optional manifest URI of an alternative ad source. If set it will use this URI during the scheduled ad break instead of Google Pod Serving.
+- `backdropURI`: Optional URI containing the back-drop to be used for this ad break.
+- `backdropURIGamProperties`: Optional property with configuration values for a dynamic back-drop loaded via GAM to be used for this ad break. This property has priority over `backdropURI`.
```

Comment:
nitpick: @tjtheo Same note on removing the hyphens in the two "back-drops" here.